tests/test_pipeline_images.py

```diff
@@ -249,9 +252,47 @@ def test_thumbs(self):
         custom_thumbs = {'small': (50, 50), 'big': (270, 270)}
         another_pipeline = ImagesPipeline.from_settings(Settings({'IMAGES_STORE': self.tempdir,
                                                                 'IMAGES_THUMBS': custom_thumbs}))
-        self.assertEqual(self.pipeline.thumbs, self.default_settings.get('IMAGES_THUMBS'))
+        default = self.pipeline.THUMBS
+        self.assertEqual(self.pipeline.thumbs, self.default_settings.get('IMAGES_THUMBS', default))
         self.assertEqual(another_pipeline.thumbs, custom_thumbs)
 
+    def test_class_attrs_preserved(self):
+
+        class UserDefinedImagePipeline(ImagesPipeline):
+            MIN_WIDTH = 1000
+
+        # If image settings are not defined values are taken from class attributes.
+        pipeline = UserDefinedImagePipeline.from_settings(Settings({"IMAGES_STORE": self.tempdir}))
+        self.assertEqual(pipeline.min_width, 1000)
+
+    def test_class_attrs_preserved_if_only_global_settings_defined(self):
+
+        class UserDefinedImagePipeline(ImagesPipeline):
+            MIN_WIDTH = 1000
+
+        settings = {
+            "IMAGES_STORE": self.tempdir,
+            "IMAGES_MIN_WIDTH": 90
+        }
+
+        # If image settings are defined they override class attributes.
+        pipeline = UserDefinedImagePipeline.from_settings(Settings(settings))
+        self.assertEqual(pipeline.min_width, 1000)
```

Comment:
good catch. Comment is outdated, it was relevant in previous version before this new update I pushed today. I'll remove it